### DISCUSSION TITLE
0.1.1 Beta

### DISCUSSION THREAD

**Discussion by Picorims** [Picorims/wav2bar#7] [General]
Note that the user interface is very Work In Progress at this moment (don't hesitate to give feedback). If you find any bugs please report them on GitHub.
This release won't include a Windows Installer because this one is not yet ready for production.

# Changelog
## Added
- A user with no configured paths for FFmpeg and/or FFprobe are now prompted with an additional tutorial
on how to install FFmpeg and FFprobe with Wav2Bar.

## Changed
- Replaced "open recommended folder" button for FFmpeg in the settings by a "open help" button
that gives instructions on how to install FFmpeg and FFprobe for Wav2Bar.

## Removed
- Removed the menu bar at the top of framed windows, that has absolutely no use in the application.

## Fixed
- Particle speed smoothing is now applied at export as well.

# Known isssues
- none.

# Note on Betas
Wav2Bar is currently in beta, which means that things actively change (UI, saves, etc.)! Please **backup your saves**, especially when updating Wav2Bar.

# Note for Linux
Right now Linux is not very well supported, but I plan to make it better in the future. For now you should be able to use Wav2Bar without much problem by downloading and running the source code (see the README or the wiki). There are some predefined (but not tested) configs to make some linux packages that you can try.

# Note for Mac
Mac is not supported at this moment, but you can give it a try using the source code (see the README or the wiki).

# Support
For bug reports, please use the issues section of GitHub. For other support, use the discussions section or go over to my Discord server (https://discord.gg/EVGzfdP)

<hr /><em>This discussion was created from the release <a href='https://github.com/Picorims/wav2bar/releases/tag/v0.1.1'>0.1.1 Beta</a>.</em>

**Comment by qaq3326** [comment]
fill the tank; come on

**Reply by Picorims** [reply]
I am currently busy with university and exams, please be patient. More info here : https://picorims.github.io/wav2bar-blog/public/2021/05/29/beta-0-1-and-general-update/
